--- a/pyproject.toml	2023-10-07 09:50:18.767805660 +0200
+++ b/pyproject.toml	2023-10-07 09:51:15.615432551 +0200
@@ -12,7 +12,7 @@
     # Included here for seamless wheel builds.
     # Packagers can choose to replace it by externally provided
     # c-blosc2 library and headers
-    "blosc2 >=2.2.8",
+    # "blosc2 >=2.2.8",
 ]
 build-backend = "setuptools.build_meta"
 # build-backend = "mesonpy"  # and replace ``setuptools`` above
@@ -72,7 +72,7 @@
     "numexpr >= 2.6.2",
     "packaging",
     "py-cpuinfo",
-    "blosc2 >= 2.2.8",
+    # "blosc2 >= 2.2.8",
 ]
 
 
