diff --git a/setup.py b/setup.py
index ab8f8e30..b79b863e 100755
--- a/setup.py
+++ b/setup.py
@@ -834,14 +834,14 @@ if __name__ == "__main__":
     cython_extfiles = get_cython_extfiles(cython_extnames)
 
     # --------------------------------------------------------------------
-    if os.name == "nt":
-        for dll_file in dll_files:
-            shutil.copy(dll_file, 'tables')
-        dll_dir = Path('tables')
-        dll_files = [dll_dir / Path(dll_file).name for dll_file in dll_files]
-
-        # Add DLL's to the final package for windows
-        data_files.append((Path("Lib/site-packages/tables"), dll_files))
+    # if os.name == "nt":
+    #     for dll_file in dll_files:
+    #         shutil.copy(dll_file, 'tables')
+    #     dll_dir = Path('tables')
+    #     dll_files = [dll_dir / Path(dll_file).name for dll_file in dll_files]
+    #
+    #     # Add DLL's to the final package for windows
+    #     data_files.append((Path("Lib/site-packages/tables"), dll_files))
 
     ADDLIBS = [hdf5_package.library_name]
 
