diff --git a/pyproject.toml b/pyproject.toml
index 64ada517..d626fc48 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -16,7 +16,7 @@ requires = [
     # Included here for seamless wheel builds.
     # Packagers can choose to replace it by externally provided
     # c-blosc2 library and headers
-    "blosc2 >= 2.3.0",
+    # "blosc2 >= 2.3.0",
     "typing-extensions >= 4.4.0",
 ]
 build-backend = "setuptools.build_meta"
@@ -78,7 +78,7 @@ dependencies = [
     "numexpr >= 2.6.2",
     "packaging",
     "py-cpuinfo",
-    "blosc2 >= 2.3.0",
+    # "blosc2 >= 2.3.0",
     "typing-extensions >= 4.4.0",
 ]
 
