diff --git a/setup.py b/setup.py
index 966be028..7152ad51 100755
--- a/setup.py
+++ b/setup.py
@@ -262,11 +262,9 @@ if __name__ == "__main__":
 
     debug = "--debug" in sys.argv
 
-    blosc2_inc, blosc2_lib = get_blosc2_directories()
-
     # Global variables
-    lib_dirs = [blosc2_lib]
-    inc_dirs = [Path("hdf5-blosc/src"), Path("hdf5-blosc2/src"), blosc2_inc]
+    lib_dirs = []
+    inc_dirs = [Path("hdf5-blosc/src"), Path("hdf5-blosc2/src")]
     optional_libs = []
     copy_libs = []
 
@@ -274,6 +272,13 @@ if __name__ == "__main__":
     default_library_dirs = None
     default_runtime_dirs = None
 
+    try:
+        blosc2_inc, blosc2_lib = get_blosc2_directories()
+        lib_dirs.append(blosc2_lib)
+        inc_dirs.append(blosc2_inc)
+    except EnvironmentError:
+        blosc2_inc = blosc2_lib = None
+
     if os.name == "posix":
         prefixes = ("/usr/local", "/sw", "/opt", "/opt/local", "/usr", "/")
         prefix_paths = [Path(x) for x in prefixes]
@@ -691,7 +696,7 @@ if __name__ == "__main__":
     # Try to locate the compulsory and optional libraries.
     lzo2_enabled = False
     compiler = new_compiler()
-    if not BLOSC2_DIR:
+    if not BLOSC2_DIR and blosc2_inc is not None:
         # Inject the blosc2 directory as detected
         BLOSC2_DIR = os.path.dirname(blosc2_inc)
     for (package, location) in [
